apis/containerattached/v1beta1/containerattachedcluster_types.go

```diff
@@ -26,8 +26,7 @@ var ContainerAttachedClusterGVK = GroupVersion.WithKind("ContainerAttachedCluste
 // +kcc:proto=google.cloud.gkemulticloud.v1.AttachedCluster
 type ContainerAttachedClusterSpec struct {
 	/* The ID of the project in which the resource belongs. If it is not provided, the provider project is used. */
-	// +optional
-	ProjectRef *refs.ProjectRef `json:"projectRef,omitempty"`
+	ProjectRef *refs.ProjectRef `json:"projectRef"`
```

Comment:
Mind elaborating this change?

Reply:
In the existing tf-based resource, this field is required even though the comment implies it is optional.
In my previous PR I made this optional to match the comment, but you called this out as a change in behavior, so I said I'd fix it. This is just putting everything back the way it was.

Reply:
Sorry I didn't make it clear. I mean the removal of the `omitempty`. I'm wondering why you make this change (because this could be an interesting topic!)